data/assets/scene/solarsystem/missions/messenger/mercurymagnetosphere.asset

```diff
@@ -0,0 +1,59 @@
+-- mercurymagnetosphere.asset
+
+local assetHelper = asset.require('util/asset_helper')
+
+local localFolder = asset.syncedResource({
+    Name = "Messenger Magnetosphere",
+    Type = "HttpSynchronization",
+    Identifier = "mercury_magnetosphere",
+    Version = 1
+})
+
+
+local MercuryRadius = 2.4397E6
+
+local Magnetosphere = {   
+    Name = "Mercury Magnetosphere",
+    Identifier = "MercuryMagnetosphere",
+    Parent = "MercuryBarycenter",
+    SceneRadius = 0.8E+5,
+    Renderable = {
+        Type = "RenderableTimeVaryingVolume",
+        SourceDirectory = localFolder,
+        TransferFunction = localFolder .. "/transferfunction.txt",
+        Variable = "rho",
+        StepSize = "0.003",
+        Dimensions = {64, 64, 64},
+        GridType = "Cartesian",
+        SecondsBefore = 24*60*60*365*100,
+        SecondsAfter = 24*60*60*365*100,
+        Enabled = false
+    },
+    Transform = {
+        Translation = {
+            Type = "SpiceTranslation",
+            Target = "MERCURY",
+            Observer = "MERCURY BARYCENTER",
+        },
+        Rotation = {
+            Type = "SpiceRotation",
+            SourceFrame = "MERCURYSE",
```

Comment:
SE is correct?

Reply:
Nevermind, just saw it

Reply:
well, I guess in that the magnetosphere had the correct orientation in regards to the planet and the sun. and none of the science people said anything to the contrary.